flytekit/clis/sdk_in_container/register.py

```diff
@@ -138,34 +146,38 @@ def register(
     cli_logger.debug(
         f"Running pyflyte register from {os.getcwd()} "
         f"with images {image_config} "
-        f"and image destinationfolder {destination_dir} "
+        f"and image destination folder {destination_dir} "
         f"on {len(package_or_module)} package(s) {package_or_module}"
     )
 
     # Create and save FlyteRemote,
     remote = get_and_save_remote_with_click_context(ctx, project, domain)
 
-    # Todo: add switch for non-fast - skip the zipping and uploading and no fastserializationsettings
-    # Create a zip file containing all the entries.
     detected_root = find_common_root(package_or_module)
     cli_logger.debug(f"Using {detected_root} as root folder for project")
-    zip_file = fast_package(detected_root, output, deref_symlinks)
+    fast_serialization_settings = None
 
-    # Upload zip file to Admin using FlyteRemote.
-    md5_bytes, native_url = remote._upload_file(pathlib.Path(zip_file))
-    cli_logger.debug(f"Uploaded zip {zip_file} to {native_url}")
+    # Create a zip file containing all the entries.
+    zip_file = fast_package(detected_root, output, deref_symlinks)
+    md5_bytes, str_digest = hash_file(pathlib.Path(zip_file))
```

Comment:
`str_digest` is not used, right? I'm surprised that the linter didn't complain about this.